src/components/Resume/ResumeCreation.jsx

```diff
@@ -0,0 +1,181 @@
+import { Button, TextField } from "@material-ui/core";
+import "./ResumeCreation.css";
+import companylogo from "../../assets/company-logo.png";
+import { useFormik } from "formik";
+import { PostResume } from "./ResumeCreationApi.js";
+import { Link, useHistory } from "react-router-dom";
+
+const validate = (values) => {
+  const errors = {};
+
+  if (!values.firstname) {
+    errors.firstname = "Name field is required";
+  } else if (values.firstname.length > 15) {
+    errors.firstname = "Please enter a name with 10 characters or less";
+  }
+  if (!values.lastname) {
+    errors.lastname = "Lastname field is required";
+  } else if (values.lastname.length > 15) {
+    errors.lastname = "Please enter a lastname with 10 characters or less";
+  }
+  if (!values.email) {
+    errors.email = "Email field is required";
+  } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(values.email)) {
+    errors.email = "Invalid email address";
+  }
+  if (!values.phone) {
+    errors.phone = "Phone number field is required";
+  } else if (values.phone < 60000000 || values.phone >= 80000000) {
+    errors.phone = "Please enter a valid phone number with 8 characters";
+  }
+  if (!values.summary) {
+    errors.summary = "Bio description field is required";
+  } else if (values.summary.length > 150) {
+    errors.summary = "Please enter a description with 150 characters or less";
+  }
+  return errors;
+};
+
+function ResumeCreation() {
+  const history = useHistory();
+  const formik = useFormik({
+    initialValues: {
+      firstname: "",
+      lastname: "",
+      email: "",
+      phone: "",
+      summary: "",
+    },
+    validate,
+    onSubmit: (values) => {
+      PostResume(values).then((result) => {
+        if (!result) {
+          history.push("/resumecreation");
+        } else {
+          alert("Success! Welcome to TeamUp!");
+          return result;
+        }
```

Comment:
Could you try changing this to async/await? It's better to use async/await ove promises